src/sdk/namespace_fs.js

```diff
@@ -889,6 +891,19 @@ class NamespaceFS {
         }
     }
 
+    async _delete_object_empty_path(obj, fs_account_config) {
+        let dir = path.dirname(obj);
```

Comment:
seems to me that you need to add error handling here since readdir and rmdir can fail,
also need to think if this should fails the whole function or not.

Reply:
Sure, I will wrap it with try and catch. 
I Will prompt a warning but will not fail the entire delete.
In this situation, we will level the path there